packages/webapp/src/components/lists/ReportList/reports/ClientReport/useClientReportColumns.tsx

```diff
@@ -31,6 +31,8 @@ export function useClientReportColumns(
 				itemClassName: styles.itemCell,
 				name: t('clientList.columns.name'),
 				onRenderColumnHeader(key, name, index) {
+					console.log('on render name col header', key, name, index) // eslint-disable-line
```

Comment:
Quick question @devan-huapaya - Do we want to keep this log in here, or can this be removed?